src/array.js

```diff
@@ -73,7 +83,12 @@ function removeFalsyValues(arr) {
  *    [ 'angular', 'react', 'ember' ] => [ 7, 5, 5 ]
  */
 function getStringsLength(arr) {
-	throw new Error("Not implemented");
+	let arr2=[];
+	for(let i=0;i<arr.length;i++)
+	{
+		arr2.push(arr[i].length);
+	}
+	return arr2;
```

Comment:
Try using the `.map` method.

Reply:
Oh Thank you for suggesting it bhaiya. Actually i was not used to it. From now I'll use it.
Aise hi hai na bhaiya:?

function getStringsLength(arr) {
    let mapping = arr.map(str => str.length);
    return mapping;
}

Reply:
Naah.
```ts
return arr.map((ele) => ele.length);
```